system_tests/das_test.go

```diff
@@ -379,13 +379,13 @@ func TestDASComplexConfigAndRestMirror(t *testing.T) {
 		// L1NodeURL: normally we would have to set this but we are passing in the already constructed client and addresses to the factory
 		RequestTimeout: 5 * time.Second,
 	}
-	l2clientC, nodeC := Create2ndNodeWithConfig(t, ctx, nodeA, l1stack, &l2info.ArbInitData, l1NodeConfigC)
+	l2clientC, _, l2stackC := Create2ndNodeWithConfig(t, ctx, nodeA, l1stack, &l2info.ArbInitData, l1NodeConfigC)
 
 	checkBatchPosting(t, ctx, l1client, l2clientA, l1info, l2info, big.NewInt(1e12), l2clientB, l2clientC)
 
-	nodeA.StopAndWait()
-	nodeB.StopAndWait()
-	nodeC.StopAndWait()
+	l2stackA.Close()
+	l2stackB.Close()
+	l2stackC.Close()
```

Comment:
There are a few places where we switched from stopping the node to closing the stack, but I think we might need to do both. It doesn't seem like the node is attached to the stack in the current test harness (though maybe we should change that instead).

Reply:
Ah I assumed it was. I think probably consolidating on attaching it to the stack and using that for stopping and starting makes sense

Reply:
Nevermind, it's attached by the node itself with `arbNodeLifecycle`